assignment01-voronoi/worldgen-voronoi.c

```diff
@@ -0,0 +1,224 @@
+#include <stdio.h>
+#include <unistd.h>
+#include <time.h>
+#include <stdlib.h>
+#include "tiles.h"
+#include <math.h>
+
+
+#define GRID_HEIGHT 24-3
+#define GRID_WIDTH 80
+#define SEED_NUM 20
+
+static char gridMatrix[GRID_HEIGHT][GRID_WIDTH];
+static int leftX, leftY, topX, topY, botX, botY, rightX, rightY;
+
+typedef struct Seed {
+    unsigned int x;
+    unsigned int y;
+    char tile;
+} Seed;
+
+
+void init_world() 
+{   
+    for(int i=0; i < GRID_HEIGHT; i++) {
+       	gridMatrix[i][0] = mountainTile.ascii;
+	    gridMatrix[i][GRID_WIDTH-1] =  mountainTile.ascii;
+    }
+    // TODO maybe better way to impliment this but oh well/ or move to its own func
+    topX = 1 + (rand() % (GRID_HEIGHT-1));
+    topY = 0;
+    gridMatrix[topX][topY] = path.ascii;
+    botX = 1 + (rand() % (GRID_HEIGHT-1));
+    botY = GRID_WIDTH - 1;
```

Comment:
why -1?

Reply:
I did -1 to make it 79 at max so we aint putting shit on da border